packages/progress-bar/src/ProgressBar/ProgressBar.tsx

```diff
@@ -118,6 +124,7 @@ export function ProgressBar(props: ProgressBarProps) {
       {screenReaderMessage && (
```

Comment:
I believe this will still render even if message is `''` since that's still a truthy value